### ISSUE TITLE
add ability to report projects for using AI generated artwork

### ISSUE THREAD

**Issue by Thinkseal** [modrinth/code#1331]
### Please confirm the following.

- [X] I checked the [existing issues](https://github.com/modrinth/code/issues) for duplicate feature requests
- [X] I have checked that this feature request is not on our [roadmap](https://roadmap.modrinth.com)

### What parts of Modrinth is your feature request related too?

Website

### Is your suggested feature related to a problem? Please describe.

Some content on modrinth uses AI generated artwork in its icon or gallery, this is a problem because most it not all AI artwork generators have been trained on at least some artwork that they (the creators of that AI) do not have the right to use which would make it count as copyright infringement of the original creators of the artwork. In addition due to the fact that almost all of the AI models used for generating artwork are attempting to just follow what they are given in their training data they are almost always copying it in many ways a lot of which would not fall under fair use.

### Describe the solution you'd like

The solution that I would like for this is the ability to report projects for using AI generated artwork in their gallery or as their icon and if that if their project is concluded to have a generated artwork in its gallery or icon, then it should be set to unlisted until the creator removed the AI generated artwork from it and then goes through the review process again.

### Describe alternatives you've considered

The first alternative I have considered is allowing this on modrinth and potentially breaking multiple peoples copyright for their work.
Another alternative I considered was allowing reporting under copyright infringement for AI generated artwork, but I believe that would be yeah, ambiguous enough that people wouldn't end up actually reporting projects for having AI generated artwork in them.
A third alternative I considered was instead of making the projects unlisted making them private until they remove the AI generated artwork from them and go through the review process again.
The fourth alternative that I considered was making it so that if someone's project is considered have AI generated content in it then instead of needing to go through the review process again, it would just need to go through a shorter one where it's just checked to make sure they actually removed the AI generated contact.

### Additional context

I believe that AI generated artwork could become a real problem for modrinth, and it would be better to solve it sooner rather than later.
I also believe that you would be better to if it is uncertain if artwork was generated by AI to assume that it was not to have as low of a false positive rate as possible.

**Comment by Prospector** [comment]
AI generated content is not strictly forbidden per our rules, but if it is in violation of copyright then it's reportable under the existing copyright report reason

**Comment by Thinkseal** [comment]
I feel like it should be at least discouraged, explicitly within the rules to lower the amount of AI generated content on modrinth most of the time is low quality.

**Comment by worldwidepixel** [comment]
I think that this would require a general Modrinth policy change on generative AI content.

**Comment by Xfrtrex** [comment]
I agree with worldwidepixel, while yes this is a big issue, this needs to be a policy change.

**Comment by Prospector** [comment]
Closing as we've explained a few times why we don't plan to ban AI-generated content, however many of the points you mentioned are already covered by our rules:
- Copyright violations are already not allowed, and can be reported via DMCA or our policy on Reuploaded Works
- Passing off AI-generated artwork in your gallery as representative of what a project does falls under our misleading description policy, and is not allowed
